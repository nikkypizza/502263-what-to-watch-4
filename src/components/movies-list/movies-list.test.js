import React from 'react';
import renderer from 'react-test-renderer';
import MoviesList from './movies-list.jsx';

const mock = [{
  title: `Snatch`,
  imgSrc: `list.jpeg`,
  linkURL: `index.html`
},
{
  title: `Snatch`,
  imgSrc: `list.jpeg`,
  linkURL: `index.html`
},
{
  title: `Snatch`,
  imgSrc: `list.jpeg`,
  linkURL: `index.html`
}];

describe(`Main Component`, () => {
  it(`renders`, () => {
    const tree = renderer.create(
        <MoviesList dataArr={mock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
