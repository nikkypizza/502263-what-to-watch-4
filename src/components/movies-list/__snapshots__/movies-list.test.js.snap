// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Component renders 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Snatch"
        height="175"
        src="list.jpeg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="index.html"
      >
        Snatch
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Snatch"
        height="175"
        src="list.jpeg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="index.html"
      >
        Snatch
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Snatch"
        height="175"
        src="list.jpeg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="index.html"
      >
        Snatch
      </a>
    </h3>
  </article>
</div>
`;
